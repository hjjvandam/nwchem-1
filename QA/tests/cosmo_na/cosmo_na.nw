start cosmo_na_dat
echo
title "Na+"
geometry
  na 0.0 0.0 0.0
end
charge +1
basis
  na library 6-31g*
end
cosmo
end
task scf energy
