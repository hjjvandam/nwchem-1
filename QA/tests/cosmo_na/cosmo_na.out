All connections between all procs tested: SUCCESS
 argument  1 = cosmo_na.nw



============================== echo of input deck ==============================
start cosmo_na
echo
title "Na+"
geometry
  na 0.0 0.0 0.0
end
charge +1
basis
  na library 6-31g*
end
cosmo
end
task scf energy
================================================================================


                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 6.1.1
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2012
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = arcen
    program       = /home/d3y133/nwchem-releases/nwchem-6.1.1-final/bin/LINUX64/nwchem
    date          = Wed Aug 28 14:56:12 2013

    compiled      = Mon_Aug_19_11:40:42_2013
    source        = /home/d3y133/nwchem-releases/nwchem-6.1.1-final
    nwchem branch = 6.1.1
    input         = cosmo_na.nw
    prefix        = cosmo_na.
    data base     = ./cosmo_na.db
    status        = startup
    nproc         =        1
    time left     =     -1s



           Memory information
           ------------------

    heap     =   13107201 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428802 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                                        Na+
                                        ---

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)



                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 na                  11.0000     0.00000000     0.00000000     0.00000000

      Atomic Mass 
      ----------- 

      na                22.989800


 Effective nuclear repulsion energy (a.u.)       0.0000000000

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000


            XYZ format geometry
            -------------------
     1
 geometry
 na                    0.00000000     0.00000000     0.00000000

                      Basis "ao basis" -> "" (cartesian)
                      -----
  na (Sodium)
  -----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  9.99320000E+03  0.001938
  1 S  1.49989000E+03  0.014807
  1 S  3.41951000E+02  0.072706
  1 S  9.46797000E+01  0.252629
  1 S  2.97345000E+01  0.493242
  1 S  1.00063000E+01  0.313169

  2 S  1.50963000E+02 -0.003542
  2 S  3.55878000E+01 -0.043959
  2 S  1.11683000E+01 -0.109752
  2 S  3.90201000E+00  0.187398
  2 S  1.38177000E+00  0.646699
  2 S  4.66382000E-01  0.306058

  3 P  1.50963000E+02  0.005002
  3 P  3.55878000E+01  0.035511
  3 P  1.11683000E+01  0.142825
  3 P  3.90201000E+00  0.338620
  3 P  1.38177000E+00  0.451579
  3 P  4.66382000E-01  0.273271

  4 S  4.97966000E-01 -0.248503
  4 S  8.43530000E-02 -0.131704
  4 S  6.66350000E-02  1.233520

  5 P  4.97966000E-01 -0.023023
  5 P  8.43530000E-02  0.950359
  5 P  6.66350000E-02  0.059858

  6 S  2.59544000E-02  1.000000

  7 P  2.59544000E-02  1.000000

  8 D  1.75000000E-01  1.000000



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 na                          6-31g*                  8       19   4s3p1d



          ---------------
          -cosmo- solvent
          ---------------
 dielectric constant -eps-  =  78.40
 charge screening approach  =   2
 screen = (eps-1)/(eps    ) =   0.98724
 -lineq- algorithm          =   1
 -bem- low  level           =   2
 -bem- high level           =   3
 -bem- from -octahedral-
 solvent radius (ang.)      =   0.000
 atomic radii = 
 --------------
    1 11.000  1.755

 solvent accessible surface
 --------------------------

 ---------- ATOMIC COORDINATES (A.U.) ------------ VDWR(ANG.) --
     1    0.00000000    0.00000000    0.00000000     1.755
 number of segments per atom =         32
 number of   points per atom =        128
 atom (   nspa,  nppa )
 ----------------------
    1 (     32,   128 )     128
 number of -cosmo- surface points =       32
 molecular surface =     38.705 angstrom**2
 molecular volume  =     22.642 angstrom**3
 G(cav/disp)       =      1.054 kcal/mol
 ...... end of -cosmo- initialization ......


                                 NWChem SCF Module
                                 -----------------


                                        Na+



  ao basis        = "ao basis"
  functions       =    19
  atoms           =     1
  closed shells   =     5
  open shells     =     0
  charge          =   1.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./cosmo_na.movecs
  use symmetry    = F
  symmetry adapt  = F


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 na                          6-31g*                  8       19   4s3p1d



 Forming initial guess at       0.0s


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:        -161.84139203

 Renormalizing density from      11.00 to     10

      Non-variational initial energy
      ------------------------------

 Total energy =    -152.531239
 1-e energy   =    -207.048349
 2-e energy   =      54.517110
 HOMO         =      -2.550013
 LUMO         =      -0.357699


 Starting SCF solution at       0.2s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-04
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-07
 ----------------------------------------------

     COSMO gas phase

 #quartets = 6.660D+02 #integrals = 3.754D+03 #direct =  0.0% #cached =100.0%


 Integral file          = ./cosmo_na.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =   1380
 No. of bits per label  =      8        No. of bits per value  =     64


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1     -161.6427062969  4.80D-01  2.07D-01      0.3
                 2     -161.6591185611  4.89D-02  1.94D-02      0.3
                 3     -161.6592883162  8.44D-04  3.21D-04      0.3
                 4     -161.6592883670  9.05D-07  5.74D-07      0.3
     COSMO solvation phase

              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1     -161.8081382878  2.35D-03  1.22D-03      0.3
                 2     -161.8081382767  2.10D-06  1.22D-06      0.3
 
                  COSMO solvation results
                  -----------------------
 
                 gas phase energy =      -161.6592883670
                 sol phase energy =      -161.8081382767
 (electrostatic) solvation energy =         0.1488499097 (   93.40 kcal/mol)


       Final RHF  results 
       ------------------ 

         Total SCF energy =   -161.808138276716
      One-electron energy =   -221.768241319448
      Two-electron energy =     63.085727263471
 Nuclear repulsion energy =      0.000000000000

        Time for solution =      0.2s


             Final eigenvalues
             -----------------

              1      
    1  -40.4830
    2   -2.7871
    3   -1.5094
    4   -1.5094
    5   -1.5094
    6    0.0305
    7    0.0580
    8    0.0580
    9    0.0580
   10    0.1573
   11    0.2557
   12    0.2557
   13    0.2557
   14    0.5204
   15    0.5204

                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------

 Vector    2  Occ=2.000000D+00  E=-2.787102D+00
              MO Center=  6.1D-17,  1.0D-16, -1.7D-16, r^2= 2.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.028271  1 Na s                  1     -0.245818  1 Na s          

 Vector    3  Occ=2.000000D+00  E=-1.509446D+00
              MO Center= -1.6D-17,  2.2D-17,  1.7D-16, r^2= 2.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.942459  1 Na pz                 4      0.247542  1 Na py         
     3     -0.219673  1 Na px         

 Vector    4  Occ=2.000000D+00  E=-1.509446D+00
              MO Center=  2.8D-17, -1.1D-16,  3.7D-17, r^2= 2.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.943910  1 Na py                 5     -0.285138  1 Na pz         
     3     -0.159662  1 Na px         

 Vector    5  Occ=2.000000D+00  E=-1.509446D+00
              MO Center= -5.5D-17, -1.1D-17,  3.9D-18, r^2= 2.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.961256  1 Na px                 4      0.213351  1 Na py         
     5      0.168017  1 Na pz         

 Vector    6  Occ=0.000000D+00  E= 3.051388D-02
              MO Center=  2.0D-14, -1.6D-14, -2.8D-14, r^2= 1.1D+01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    10      1.583403  1 Na s                  6     -1.057817  1 Na s          
     2     -0.210845  1 Na s                 14      0.204996  1 Na dxx        
    17      0.204996  1 Na dyy               19      0.204996  1 Na dzz        

 Vector    7  Occ=0.000000D+00  E= 5.804728D-02
              MO Center= -3.9D-15,  4.7D-15,  3.3D-14, r^2= 1.5D+01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    13      1.095141  1 Na pz                 9     -0.178891  1 Na pz         
    12      0.157016  1 Na py         

 Vector    8  Occ=0.000000D+00  E= 5.804728D-02
              MO Center= -1.5D-14,  1.3D-14, -3.9D-15, r^2= 1.5D+01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11      0.833843  1 Na px                12     -0.710845  1 Na py         
    13      0.199783  1 Na pz         

 Vector    9  Occ=0.000000D+00  E= 5.804728D-02
              MO Center= -5.2D-16, -1.1D-15, -6.9D-17, r^2= 1.5D+01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    12      0.842944  1 Na py                11      0.727113  1 Na px         

 Vector   10  Occ=0.000000D+00  E= 1.573475D-01
              MO Center=  7.9D-15, -7.5D-15, -9.5D-15, r^2= 1.1D+01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      4.222902  1 Na s                 10     -2.228397  1 Na s          
    14     -0.757796  1 Na dxx               17     -0.757796  1 Na dyy        
    19     -0.757796  1 Na dzz        

 Vector   11  Occ=0.000000D+00  E= 2.556728D-01
              MO Center= -3.8D-15, -2.6D-15, -3.8D-16, r^2= 6.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.319454  1 Na px                11     -0.749250  1 Na px         
     8      0.354843  1 Na py                 3     -0.208152  1 Na px         
    12     -0.201497  1 Na py         

 Vector   12  Occ=0.000000D+00  E= 2.556728D-01
              MO Center= -1.4D-15,  6.9D-15, -1.5D-16, r^2= 6.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.319254  1 Na py                12     -0.749136  1 Na py         
     7     -0.350382  1 Na px                 4     -0.208120  1 Na py         
    11      0.198964  1 Na px         

 Vector   13  Occ=0.000000D+00  E= 2.556728D-01
              MO Center=  1.4D-15, -9.5D-16,  4.4D-15, r^2= 6.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      1.363429  1 Na pz                13     -0.774221  1 Na pz         
     5     -0.215089  1 Na pz         

 Vector   14  Occ=0.000000D+00  E= 5.203961D-01
              MO Center= -1.6D-16,  3.0D-16,  3.1D-16, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    18      1.591746  1 Na dyz               15     -0.650381  1 Na dxy        
    16      0.208208  1 Na dxz        

 Vector   15  Occ=0.000000D+00  E= 5.203961D-01
              MO Center= -6.5D-16,  8.7D-17,  1.3D-16, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    16      1.719270  1 Na dxz               18     -0.203229  1 Na dyz        


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
           0.000000000000           0.000000000000           0.000000000000
           0.000000000000           0.000000000000           0.000000000000
           0.000000000000           0.000000000000           0.000000000000

  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 Na  11    10.00   2.00  2.00  5.99  0.00  0.01  0.00  0.00  0.00

       Multipole analysis of the density wrt the origin
       ------------------------------------------------

     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0      1.000000      0.000000     11.000000

     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000
     1   0 0 1      0.000000      0.000000      0.000000

     2   2 0 0     -2.153342      0.000000      0.000000
     2   1 1 0      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -2.153342      0.000000      0.000000
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -2.153342      0.000000      0.000000


 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.3s     wall:        0.4s
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  195      195     1566      890      794        0        0        0     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             7.84e+05 4.97e+05 1.23e+05 0.00e+00 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 39432 bytes
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        17	        14
	current total bytes		         0	         0
	maximum total bytes		   1062640	  22509304
	maximum total K-bytes		      1063	     22510
	maximum total M-bytes		         2	        23


                                NWChem Input Module
                                -------------------





                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
    J. Autschbach, F. Aquino, J. Mullin, P. Nichols, S. Hirata, M. T. Hackler,
   Y. Zhao, P.-D. Fan, R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann,
    J. Nieplocha, V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen,
      M. Swart, Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby,
        E. Brown, G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, G. Sandrone,
       M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        0.3s     wall:        0.4s
